locales/fr_FR/pcsx2_Main.po

```diff
@@ -1733,8 +1729,7 @@ msgstr "Pause/Reprendre"
 
 #: pcsx2/gui/MainFrame.cpp:664
 msgid "No emulation state is active; cannot suspend or resume."
-msgstr ""
-"Aucune émulation en cours : impossible d'interrompre ou de reprendre "
+msgstr "Aucune émulation en cours : impossible d'interrompre ou de reprendre "
```

Comment:
Missing period and remove extra space at the end

Reply:
French language has spaces before and after theses characters:  ? ! : ; % " + - / * =
See: https://www.blogdumoderateur.com/regles-ponctuation-typographie/